app/models/hmcts_common_platform/court_application_summary.rb

```diff
@@ -83,6 +87,7 @@ def to_builder
         summary.hearing_summary hearing_summary.map(&:to_json)
         summary.subject_summary subject_summary.to_json
         summary.judicial_results judicial_results.map(&:to_json)
+        summary.maat_id maat_id
```

Comment:
I wonder if the fact that `maat_id` might be nil could be confusing,
More precisely, `maat_id` might be present but not linked, and that would set maat_id to nil 
Should we call the attribute linked_maat_id ?
Or maybe add a linked attribute?
🤔